namespace cpp entity
namespace py ccentity.mail_system
namespace as3 com.dreamfire.sanguo.data.definition.entity

/////////////////////////////////////////////////////////
// 系统邮件子类型
enum SystemMailChildType {
  MIN = 0,

  /////////////////////////
  // 实体类 0 ~ 99
  ACTOR_BEGIN = 0,
  /////////////////////////

  /////////////////////////
  // 道具类 100 ~ 199
  ITEM_BEGIN = 100,
  /////////////////////////

  /////////////////////////
  // 技能类 200 ~ 299
  SKILL_BEGIN = 200,
  /////////////////////////

  /////////////////////////
  // BUFF类 300 ~ 399
  BUFF_BEGIN = 300,
  /////////////////////////
  
  /////////////////////////
  // 场景类 400 ~ 499
  SCENE_BEGIN = 400,
  /////////////////////////

  /////////////////////////
  // 英灵类 500 ~ 599
  SOU_BEGI = 500,
  /////////////////////////

  /////////////////////////
  // 任务类 600 ~ 699
  TASK_BEGIN = 600,
  /////////////////////////

  /////////////////////////
  // 属性类 700 ~ 799
  ATTR_BEGIN = 700,
  /////////////////////////

  /////////////////////////
  // 聊天类 800 ~ 899
  CHAT_BEGIN = 800,
  /////////////////////////

  /////////////////////////
  // 邮件类 900 ~ 999
  MAIL_BEGIN = 900,
  /////////////////////////

  /////////////////////////
  // 帮会类 1000 ~ 1099
  GUILD_BEGIN = 1000,
  GUILD_DISMISS = 1001,
  /////////////////////////

  MAX,
}
